CryptoWallet/Scenes/Home/Wallet/AssetCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AssetCell.swift
+//  CryptoWallet
+//
+//  Created by Phan Dinh Van on 4/5/19.
+//  Copyright © 2019 Phan Dinh Van. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+final class AssetCell: UITableViewCell {
+    @IBOutlet private weak var assetLogoImageView: UIImageView!
+    @IBOutlet private weak var assetNameLabel: UILabel!
+    @IBOutlet private weak var assetPriceLabel: UILabel!
+    @IBOutlet private weak var asset24hChangeLabel: UILabel!
+    @IBOutlet private weak var assetAmountLabel: UILabel!
+    @IBOutlet private weak var assetValueLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    override func draw(_ rect: CGRect) {
+        super.draw(rect)
+        assetLogoImageView.setBorder(cornerRadius: assetLogoImageView.frame.width/2,
+                                     borderWidth: 1,
+                                     borderColor: .lightGray)
+    }
+    
+    func setCellValue(_ assetInfo: AssetInfo) {
+        assetLogoImageView.do {
+            $0.image = assetInfo.assetLogo
+            $0.contentMode = .scaleAspectFit
+            $0.clipsToBounds = true
+        }
+        assetNameLabel.do {
+            $0.text = assetInfo.assetName
+        }
+        assetPriceLabel.do {
+            $0.text = "$\(assetInfo.assetPrice)"
+        }
+        asset24hChangeLabel.do {
+            if assetInfo.asset24hChange < 0 {
+                $0.textColor = .red
```

Comment:
```suggestion
                $0.textColor = assetInfo.asset24hChange < 0 ? .red : colorName
```
đoạn này có thể rút gọn như sau
và phần màu thì có thể thêm vào extension của UIColor để các lần sau có thể gọi ra tái sử dụng